libusb/libusb.h

```diff
@@ -523,7 +524,10 @@ enum libusb_bos_type {
 	LIBUSB_BT_SS_USB_DEVICE_CAPABILITY = 0x03,
 
 	/** Container ID type */
-	LIBUSB_BT_CONTAINER_ID = 0x04
+	LIBUSB_BT_CONTAINER_ID = 0x04,
+
+	/** Platform descriptor */
+	LIBUSB_BT_PLATFORM_DESCRIPTOR = 0x05,
```

Comment:
As much as I would love to take the trailing comma there is a reason we don't have these. Microsoft for a long time neglected their C compiler and it remained only ISO-C89 compliant for a long time. The trailing comma is a ISO-C99 feature and would break older Microsoft compilers. I don't care at all but we should run this by someone who cares about windows before we move libusb.h to ISO-C99.

Reply:
Thanks for pointing out the ISO-C89 compliance, I don't think we have that mentioned anywhere. The comma would also be inconsistent with the other enumerations in that file. I don't see it as useful to break compatibility for an unnecessary comma.